feature_map.gemspec

```diff
@@ -27,17 +27,17 @@ Gem::Specification.new do |spec|
   spec.files = Dir['README.md', 'lib/**/*', 'bin/**/*']
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'code_ownership'
-  spec.add_dependency 'faraday'
-  spec.add_dependency 'packs-specification'
+  spec.add_dependency 'code_ownership', '~> 1.34'
```

Comment:
If any dependencies or development dependencies have an open-ended version assigned, the `gem build feature_map.gemspec` command prints a warning message like the following:

```
WARNING:  open-ended dependency on faraday (>= 0) is not recommended
  use a bounded requirement, such as '~> x.y'
```